package.json

```diff
@@ -60,6 +60,8 @@
     "libp2p-tcp": "~0.11.0",
     "libp2p-webrtc-star": "~0.13.0",
     "libp2p-websockets": "~0.10.1",
+    "libp2p-websocket-star": "0.3.0-rc4",
+    "libp2p-websocket-star-signal": "0.3.0-rc4",
```

Comment:
Haven't seen that we are doing RC versions anywhere else, can we release stable versions of these?

Reply:
Fixed